### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by Discocowboy** [DSCoalition/dsc-11ty-cms#81]
**Make the red links a bit darker**
- Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.nav-link[target=“_blank”][rel=“noopener”])
- Tags: Accessibility, WCAG 1.4.3, color-contrast
- Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)
- Target application: Digital Services Coalition - https://staging.digitalservicescoalition.org/
- Element path: `.desktop\:pb-10 > nav > .hidden.navbar-links.desktop\:visible > ul > li:nth-child(6) > .nav-link[target=“_blank”][rel=“noopener”]`
- Snippet: `<a href=“https://shop.digitalservicescoalition.org/” target=“_blank” rel=“noopener” class=“nav-link”>Shop</a>`

_Fix any of the following:_
- Element has insufficient color contrast of 4.39 (foreground color: #EC193F, background color: #FFFFFF, font size: 10.8pt (14.4px), font weight: normal). Expected contrast ratio of 4.5:1
- Environment: Microsoft Edge version 117.0.2045.31
![1](https://github.com/DSCoalition/dsc-11ty-cms/assets/12417007/793b4f4e-a632-4569-8aa6-2483f7076a51)

**The Gray needs additional contrast**
**Company names can be bold**
- Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.text-grayLight.text-\[0\.8rem\])
- Tags: Accessibility, WCAG 1.4.3, color-contrast
- Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)
- Target application: Digital Services Coalition - https://staging.digitalservicescoalition.org/story/
- Element path:` .font-serif.text-center.px-4:nth-child(2) > .text-grayLight.text-\[0\.8rem\]`
- Snippet: `<div class=“text-grayLight text-[0.8rem]“>Exygy</div>`
- Related paths: `.bg-grayLightest`

_Fix any of the following:_
- Element has insufficient color contrast of 4.19 (foreground color: #777777, background color: #F9F7F6, font size: 9.6pt (12.8px), font weight: normal). Expected contrast ratio of 4.5:1
- Environment: Microsoft Edge version 117.0.2045.31
![2](https://github.com/DSCoalition/dsc-11ty-cms/assets/12417007/615727a6-f74a-4ff7-ade7-70e67ea3688b)

**It is better to not duplicate the name. So we should change:**
`<div class=“font-serif flex flex-col items-center text-center px-4 pb-4 mx-auto desktop:pb-0 desktop:mt-4”><img src=“/assets/img/board-member-traci.jpg” alt=“Traci Walker” class=“w-full h-auto pb-6 desktop:pb-4 xl:w-[18.75rem] xl:h-[19.75rem]“><div class=“text-blue text-base font-bold”>Traci Walker</div><div class=“text-grayLight text-[0.8rem]“>Digital Services Coalition</div><div class=“text-blue text-[0.8rem]“>Executive Director</div></div>`
to:

`<div class=“font-serif flex flex-col items-center text-center px-4 pb-4 mx-auto desktop:pb-0 desktop:mt-4”><img src=“/assets/img/board-member-traci.jpg” alt=“” class=“w-full h-auto pb-6 desktop:pb-4 xl:w-[18.75rem] xl:h-[19.75rem]“><div class=“text-blue text-base font-bold”>Traci Walker</div><div class=“text-grayLight text-[0.8rem]“>Digital Services Coalition</div><div class=“text-blue text-[0.8rem]“>Executive Director</div></div>`
on

**Finally, the iframe should have a title to explain what it is:**
- Title: WCAG 4.1.2: Ensures <iframe> and <frame> elements have an accessible name (iframe[frameborder=“0”])
- Tags: Accessibility, WCAG 4.1.2, frame-title
- Issue: Ensures <iframe> and <frame> elements have an accessible name (frame-title - https://accessibilityinsights.io/info-examples/web/frame-title)
- Target application: Digital Services Coalition - https://staging.digitalservicescoalition.org/contact/
- Element path: `iframe[frameborder=“0"]`
- Snippet: `<iframe src=“https://docs.google.com/forms/d/e/1FAIpQLSfOo7ss8JtJ1mWCNdXoUNB3F-pj6OSabhmt9ZBeE6PcVQ1dDA/viewform?embedded=true” style=“height: 1020px;” frameborder=“0” marginheight=“0" marginwidth=“0” width=“100%“></iframe>`

_Fix any of the following:_
- Element has no title attribute
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element’s default semantics were not overridden with role=“none” or role=“presentation”
- Environment: Microsoft Edge version 117.0.2045.31
![3](https://github.com/DSCoalition/dsc-11ty-cms/assets/12417007/23b123c3-1ee3-400c-ba5e-8ec3de2bf74e)
